BUILDING.md

```diff
@@ -37,6 +33,7 @@ $ cmake .. <platform-specific generator parameters (see below)>
 # build step
 $ cmake --build . -- <platform-specific parallelism parameters (see below)>
 ```
+If OpenSSL is installed in a custom location, then one can pass `OPENSSL_ROOT_DIR` during the CMake configuration step. e.g.  add `-DOPENSSL_ROOT_DIR=/path/to/openssl`.
```

Comment:
I think we need sort of a combination of these two comments.  This one doesn't mention that it's _required_, and 'can pass' should probably be replaced by 'must pass', because the build will fail without it.

Reply:
I haven't mentioned this as required at this step because it is one of the dependency that one has to install and is part of prerequisites section and the reason for `can pass` is because it is only required to pass if you have not installed in the standard location.

Reply:
Well exactly - the sentence begins 'If OpenSSL is installed in a custom location', which means it's required.

Reply:
Agreed. Updated to `must pass`.